cscs-checks/microbenchmarks/spec-accel/spec.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, prg_envs):
 
         self.num_tasks = 1
         self.num_tasks_per_node = 1
-        self.time_limit = (0, 30, 0)
+        self.time_limit = '30m'
```

Comment:
Same here. I don't think they run for so long.

Reply:
@vkarak this one needs this timelimit since the run takes longer.